app/controllers/work_packages/bulk_controller.rb

```diff
@@ -92,6 +76,21 @@ def destroy
     end
   end
 
+  # Sets the `flash` notice or error based the number of work packages that did not save
+  #
+  # @param [Array, WorkPackage] work_packages all of the saved and unsaved WorkPackages
+  # @param [Array, Integer] unsaved_work_packages the WorkPackages that were not saved
+  def set_flash_from_bulk_save(work_packages, unsaved_work_packages)
+    if unsaved_work_packages.empty?
+      flash[:notice] = l(:notice_successful_update) unless work_packages.empty?
+    else
+      flash[:error] = l(:notice_failed_to_save_work_packages,
+                        count: unsaved_work_packages.size,
+                        total: work_packages.size,
+                        ids: unsaved_work_packages.map {|wp| '#' + wp.id}.join(', '))
```

Comment:
Line is too long. [85/80]<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.